dsgrid/config/dimensions.py

```diff
@@ -400,34 +416,22 @@ class AnnualTimeDimensionModel(TimeDimensionBaseModel):
 
 
 class RepresentativePeriodTimeDimensionModel(TimeDimensionBaseModel):
-    """Defines a representative time dimension"""
+    """Defines a representative time dimension."""
 
     format: RepresentativePeriodFormat = Field(
         title="format",
         description="Format of the timestamps in the load data",
     )
-    str_format: str = Field(
-        title="str_format",
-        description="Timestamp string format",
-        notes=(
-            "The string format is used to parse the timestamps provided in the time ranges."
-            "Cheatsheet reference: `<https://strftime.org/>`_.",
-        ),
-    )
```

Comment:
Are we sure this is no longer needed? Generally all this time stuff seems like it should be unrelated to supporting a one-table format?

Reply:
@elainethale You'll want to read https://github.com/dsgrid/dsgrid/pull/100 first. This PR includes those changes. In order to support TEMPO I need to refactor handling of time. Lixi is going to build on top of that.

My refactor involved separating some functionality into different files. You're welcome to voice disagreement in that PR. I hit a point where there was too much different logic in the same file.

Reply:
Thanks, Dan. That was the information I was missing.